### ISSUE TITLE
Agregar datos por grupo de riesgo

### ISSUE THREAD

**Issue by juancri** [juancri/covid19-vaccination#5]
La [información publicada por el DEIS](https://informesdeis.minsal.cl/SASVisualAnalytics/?reportUri=%2Freports%2Freports%2F1a8cc7ff-7df0-474f-a147-929ee45d1900&sectionIndex=0&sso_guest=true&reportViewOnly=true&reportContextBar=false&sas-welcome=false) incluye la "distribución de personas vacunadas según dosis por grupo de riesgo".

Quiero agregar esta información. Lo más probable es que sea un segundo archivo.

**Comment by juancri** [comment]
Aprovechando que está planificado #6, voy a hacer lo mismo para los grupos vacunados.

El siguiente paso será publicar un archivo de muestra y luego el archivo definitivo. ETA: domingo 07/02.

Sugerencias y comentarios son bienvenidos.

**Comment by juancri** [comment]
Esperando https://github.com/juancri/covid19-vaccination/issues/8

**Comment by juancri** [comment]
Esperando #10

**Comment by juancri** [comment]
En progreso

**Comment by juancri** [comment]
Tengo dudas de si vale la pena publicarlo tomando en cuenta que en el sitio del DEIS no hay datos retroactivos. Si alguien conoce -o se atreve a hacerle ingeniería reversa- al lenguaje de consulta que se utiliza para estos datos, en teoría, podríamos obtener los datos retroactivos. Ver [payloads](https://github.com/juancri/covid19-vaccination/tree/master/process/payloads).

**Comment by dna33** [comment]
Hola ¿estos payload se actualizan diariamente?

**Comment by juancri** [comment]
Hola. AFAIK, los payloads no necesitan actualizarse. Se reutilizan.

**Comment by juancri** [comment]
Ya que no existe historial, estoy pensando en ir agregando columnas diariamente. Este seria el formato:

| group                                      | dose   | 2021-02-15 | 
|--------------------------------------------|--------|------------| 
| Total                                      | First  | 1846093    | 
| Total                                      | Second | 54583      | 
| Desconocido                                | First  | 67         | 
| Desconocido                                | Second | 41         | 
| Personal de salud                          | First  | 470426     | 
| Personal de salud                          | Second | 53661      | 
| Personas en centros cerrados               | First  | 31339      | 
| Personas en centros cerrados               | Second | 866        | 
| Personas Servicios esenciales del estado   | First  | 166822     | 
| Personas Servicios esenciales del estado   | Second | 15         | 
| Personas Servicios infraestructura crítica | First  | 20314      | 
| Personas Servicios infraestructura crítica | Second | 0          | 
| Población Sana                             | First  | 24977      | 
| Población Sana                             | Second | 0          | 
| Población vulnerable                       | First  | 1132148    | 
| Población vulnerable                       | Second | 0          |

**Comment by dna33** [comment]
Parece super bien, importante:

1. procurar que los nombres de las categorías sean las que define el DEIS
2. no se qué más :)

**Comment by juancri** [comment]
@dna33,

> procurar que los nombres de las categorías sean las que define el DEIS

Sí. Lo programé para que los nombres (excepto por los especiales "Total" y "Desconocido") vienen desde la web del DEIS.

> PD: ¿qué ocupas para programar los js? Clonamos tu repo y en general nuestro equipo sabe python, la mayoría con un Setup ubunty + pycharm... pero hay gente que quiere aprender, alguna recomendación?

Para editar, [vscode](https://code.visualstudio.com/). Avísame si necesitan más info.

**Comment by juancri** [comment]
Publicado aquí:

https://github.com/juancri/covid19-vaccination/blob/master/output/chile-vaccination-groups.csv

**Comment by dna33** [comment]
Incorporado al GitHub de Minciencia como input, publicando resultados. Muchas gracias.

**Comment by juancri** [comment]
Ver https://github.com/juancri/covid19-vaccination/issues/16

**Comment by juancri** [comment]
El archivo por grupo de riesgos [chile-vaccination-groups.csv](https://github.com/juancri/covid19-vaccination/blob/master/output/chile-vaccination-groups.csv) se está actualizando nuevamente desde el 07/03/2020.

**Comment by juancri** [comment]
Este archivo ya no se actualizará debido a que no está disponible en el reporte del DEIS
